inst/rmd/gimapQCTemplate.Rmd

```diff
@@ -0,0 +1,103 @@
+---
+title: "QC Report"
+date: "`r Sys.Date()`"
+author: "`r paste0('gimap (v', utils::packageVersion('gimap'), ')')`"
+output: 
+  html_document:
+    theme: spacelab
+    toc: true
+    toc_float: TRUE
+    df_print: kable
+    code_folding: show
+  pdf_document:
+    toc: true
+  always_allow_html: yes
+  editor_options:
+    chunk_output_type:console
+---
+
+<style>
+  tbody tr:nth-child(odd){
+    background-color: #D6E0F5;
+}
+</style>
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
+knitr::opts_knit$set(progress = TRUE, verbose = FALSE)
+```
+
+
+# Unfiltered/raw data
+
+## Counts CDF
+
+```{r}
+list_of_qc_things$counts_cdf
```

Comment:
How is this dataset passed in? Can we have the passing in of the data be more explicit using params? https://rmarkdown.rstudio.com/lesson-6.html? I think this might make it more flexible for re-rendering.

Reply:
I can be more explicit about it, but it uses the [qc_generateReport.R function](https://github.com/FredHutch/gimap/blob/ki_qc/R/qc_generateReport.R) and seems to be inheriting the list that I [passed to that function in the running QC function](https://github.com/FredHutch/gimap/blob/fc3090334b7cfe32e95f1481939eb020390fecef/R/01-qc.R#L99)

Reply:
Is it only inheriting that because you'd ran it in the same R session though? We need to know that if someone runs this in a separate R session it would still find the data.

Reply:
I don't currently have it set up to export the function/such that it can be run separately from the overall QC steps. Are you envisioning that it can? That someone can run QC, save the "list of things"/input in an RDA and then load that RDA and generate the report in another session?

Reply:
If I understand what you are asking, I always kind of default on exporting the individual functions just in case a user has need to run them separately that we don't anticipate. Exporting the functions doesn't really take any extra work.